setup_build_env.sh

```diff
@@ -10,6 +10,7 @@ cp -r $SOURCE_DIR/thirdparty $BUILD_DIR
 cd $BUILD_DIR
 ./thirdparty/download_thirdparty.sh
 ./thirdparty/build_thirdparty.sh
+./thirdparty/versions.sh
```

Comment:
Yep, thanks. It is sourced in the other scripts but you'll want `source setup_build_env.sh` to set the right env variables